src/derivation.js

```diff
@@ -335,6 +346,20 @@ export const runDerivationScheme = (
     .replace("<node>", String(opts.node || 0))
     .replace("<address>", String(opts.address || 0));
 
+// execute the derivation on the account part of the scheme
+export const runAccountDerivationScheme = (
+  scheme: string,
+  currency: { coinType: number },
+  opts: {
+    account?: number | string,
+  } = {}
+) =>
+  runDerivationScheme(scheme, currency, {
+    ...opts,
+    address: "_",
```

Comment:
What's the logic for passing an underscore here 🤔

Reply:
We only want the account part of the derivation path, so I used _ as a way to identify which part is not account based. It's a bit overkill but it works